charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -11,7 +11,7 @@ java:
   readinessDelay: 45
   environment:
     PDF_API_URL: "http://rpe-pdf-service-aat.service.core-compute-aat.internal"
-    CORE_CASE_DATA_API_URL: "http://${SERVICE_NAME}-ccd-data-store-api"
+    CORE_CASE_DATA_API_URL: ${CORE_CASE_DATA_API_URL}
```

Comment:
these are the same variable

Reply:
${CORE_CASE_DATA_API_URL} would point to aat if the enable_hearings tag is on the PR